app/assets/javascripts/user/app/controllers/user_subscription_preferences_controller.js

```diff
@@ -3,24 +3,60 @@ this.User.SubscriptionPreferencesController = (function() {
 
   SubscriptionPreferencesController.prototype.index = function(){
     _handleSiteSelected();
+    _handleAreaSelected('gobierto_budget_consultations');
```

Comment:
Having the three modules hardcoded doesn't scale well in the future.  I propose to add a special attribute (a `data-` or a `rel=`) and loop through them.